styles/_mixins.scss

```diff
@@ -12,4 +12,8 @@
     top: 50%;
     transform: translate(0, -50%);
   }
+}
+
+@mixin chatOpeningTransition($property) {
+  transition: $property 0.5s ease;
```

Comment:
Why make this a mixin? Do you think we'll use the same transition time and easing type more than once?

If we are going to re-use it, should it be called chatOpeningTransition?